srcpkgs/StormLib/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'StormLib'
+pkgname=StormLib
+version=9.24
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DWITH_LIBTOMCRYPT=ON"
```

Comment:
```suggestion
configure_args="-DBUILD_SHARED_LIBS=ON -DWITH_LIBTOMCRYPT=ON"
```
don't set cmake build type unless absolutely necessary, it is defined in the build style

Reply:
Done!